package/etc/conf.d/conflib/syslog/app-syslog-dell_switch_n.conf

```diff
@@ -1,33 +0,0 @@
-block parser app-syslog-dell_switch_n() {
```

Comment:
I have only one concern about that. We migrating from `syslog`  filter to `netsource`. I think it might break backward compatibility  for user who will update and don't configure it on custom port / sc4s-vps filter.